Tests/ValidateInputTest.cs

```diff
@@ -27,7 +27,7 @@ public class ValidateInputTest {
         public void ReturnTrueIfInputIsOnBoard(string input) {
             bool hasValue = this.validateInput.IsInputOnBoard(input, this.board.GameBoard);
 
-            Assert.Equal(true, hasValue);
+            Assert.True(hasValue);
```

Comment:
While I like that you changed these assertions, one thing to consider when choosing an assertion is how useful its failure message is. You should experiment with that when there is more than one assertion that might be suitable for your test.